contracts/container/contract.go

```diff
@@ -491,6 +493,96 @@ func List(owner []byte) [][]byte {
 	return list
 }
 
+// AddNextEpochNodes accumulates passed nodes as container members for the next
+// epoch to be committed using [CommitContainerListUpdate]. Results of the call
+// operation can be received via [Nodes]. This method must be called only when
+// a container list is changed, otherwise nothing should be done.
+// Call must be signed by the Alphabet nodes.
+func AddNextEpochNodes(cID interop.Hash256, publicKeys []interop.PublicKey) {
```

Comment:
How about `REP 1 IN ... REP 2 IN ...`?

Reply:
Another thought is managing a _diff_ of lists. But this can be done in future.

Reply:
> How about REP 1 IN ... REP 2 IN ...?

what do you mean? did you plan to collect signatures by every SELECT? did you plan to have container nodes iterators per SELECTion?

> Another thought is managing a diff of lists

obvious thought but i do not know if it helps. it would complicate IR and contract but i do not see real pros: container change should not be an often operation but even if it happens, a huge number of nodes can be changed at a time with no problems

Reply:
How are we going to solve https://github.com/nspcc-dev/neofs-contract/issues/413 without it?